sourcegraph/sourcegraph
# Ref panel: remove unused code and coolCodeIntel feature flag

## Description
Addresses https://github.com/sourcegraph/sourcegraph/issues/13722.
Since we're now using ref panel by default, this PR removes all unused code including `coolCodeIntel` feature flag. 

## Test plan
1. Pull the branch
2. Run the app locally `sg start web-standalone`
3. Ensure new ref panel works with/without the feature flag `coolCodeIntel`

## App preview:

- [Web](https://sg-web-og-ref-panel-new.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-zbmoqtrnos.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
